orttraining/tools/ci_test/run_bert_perf_test.py

```diff
@@ -26,10 +26,10 @@ def main():
 
     Config = namedtuple('Config', ['use_mixed_precision', 'max_seq_length', 'batch_size', 'max_predictions_per_seq'])
     configs = [
-        Config(True, 128, 64, 20),
-        Config(True, 512, 10, 80),
-        Config(False, 128, 33, 20),
-        Config(False, 512, 5, 80)
+        Config(True, 128, 76, 20),
+        Config(True, 512, 11, 80),
+        Config(False, 128, 39, 20),
+        Config(False, 512, 6, 80)
```

Comment:
Should we also add max batch size for recompute in memory footprint regression test? Could be in a separate PR.

Reply:
Yes. Will do in another PR.